homeassistant/components/ecobee/__init__.py

```diff
@@ -73,6 +78,24 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
+    # The legacy Ecobee notify.notify service is deprecated
+    # was with HA Core 2025.5.0 and will be removed with HA core 2024.7.0
```

Comment:
Is 2024/2025 swapped here?
```suggestion
    # was with HA Core 2024.5.0 and will be removed with HA core 2025.7.0
```

Reply:
Thnx